examples/progressbar_accelerometer.py

```diff
@@ -0,0 +1,231 @@
+    (4.242243, -9.844303, 1.694982),
+    (5.946802, -3.543183, 6.890055),
+    (7.028910, -4.462496, 4.199150),
+    (-1.694982, -6.655439, 6.430399),
+    (0.703849, -3.112255, 8.685585),
+    (1.340664, 4.342793, 8.053558),
+    (1.627949, 9.920914, -0.608087),
+    (6.985817, 0.517113, 7.517294),
+    (5.434477, -5.372232, 5.994682),
+    (4.165634, -6.224510, 8.082287),
+    (0.847491, -4.677959, 9.509136),
+    (3.476150, -4.812025, 7.421532),
+]
+display = board.DISPLAY
+main_group = displayio.Group(max_size=10)
+display.show(main_group)
+
+color_bitmap = displayio.Bitmap(display.width, display.height, 1)
+color_palette = displayio.Palette(1)
+color_palette[0] = 0x990099
+
+background = displayio.TileGrid(color_bitmap, pixel_shader=color_palette, x=0, y=0)
+main_group.append(background)
+
+# Accelerometer Properties. Normally accelerometer well calibrated
+# Will give a maxium output of 10 mts / s**2
+VALUES_X = (-10, 10)
+VALUES_Y = (-10, 10)
+
+# Horizontal Bar Properties
+HORIZONTAL_BAR_X_ORIGIN = 10
+HORIZONTAL_BAR_Y_ORIGIN = 30
+HORIZONTAL_BAR_WIDTH = display.width // 4
+HORIZONTAL_BAR_HEIGHT = 30
+
+# Vertical Bar Properties
+VERTICAL_BAR_HEIGHT = display.height // 4
+VERTICAL_BAR_WIDTH = 30
+
+# We create our bar displays
+left_horizontal_bar = HorizontalProgressBar(
+    (HORIZONTAL_BAR_X_ORIGIN, HORIZONTAL_BAR_Y_ORIGIN),
+    (HORIZONTAL_BAR_WIDTH, HORIZONTAL_BAR_HEIGHT),
+    min_value=0,
+    max_value=-VALUES_X[0],
+    direction=HorizontalFillDirection.RIGHT_TO_LEFT,
+)
+main_group.append(left_horizontal_bar)
+
+right_horizontal_bar = HorizontalProgressBar(
+    (HORIZONTAL_BAR_X_ORIGIN + HORIZONTAL_BAR_WIDTH, HORIZONTAL_BAR_Y_ORIGIN),
+    (HORIZONTAL_BAR_WIDTH, HORIZONTAL_BAR_HEIGHT),
+    min_value=0,
+    max_value=VALUES_X[1],
+    direction=HorizontalFillDirection.LEFT_TO_RIGHT,
+)
+main_group.append(right_horizontal_bar)
+
+
+top_vertical_bar = VerticalProgressBar(
+    (HORIZONTAL_BAR_X_ORIGIN + 2 * HORIZONTAL_BAR_WIDTH + 20, HORIZONTAL_BAR_Y_ORIGIN),
+    (VERTICAL_BAR_WIDTH, VERTICAL_BAR_HEIGHT),
+    min_value=0,
+    max_value=VALUES_Y[1],
+    direction=VerticalFillDirection.BOTTOM_TO_TOP,
+)
+main_group.append(top_vertical_bar)
+
+bottom_vertical_bar = VerticalProgressBar(
+    (
+        HORIZONTAL_BAR_X_ORIGIN + 2 * HORIZONTAL_BAR_WIDTH + 20,
+        HORIZONTAL_BAR_Y_ORIGIN + VERTICAL_BAR_HEIGHT,
+    ),
+    (VERTICAL_BAR_WIDTH, VERTICAL_BAR_HEIGHT),
+    min_value=0,
+    max_value=-VALUES_Y[0],
+    direction=VerticalFillDirection.TOP_TO_BOTTOM,
+)
+main_group.append(bottom_vertical_bar)
+
+delay = 0.5
+
+while True:
+    for val in fake__accel_data:
+        if val[0] > 0:
+            left_horizontal_bar.value = 0
+            right_horizontal_bar.value = val[0]
+        if val[0] < 0:
+            left_horizontal_bar.value = -val[0]
+            right_horizontal_bar.value = 0
+        if val[0] == 0:
+            right_horizontal_bar.value = 0
+            left_horizontal_bar.value = 0
```

Comment:
This case could be merged into one of the cases above by using, say, `val[0] >= 0`.

Reply:
solved in https://github.com/adafruit/Adafruit_CircuitPython_ProgressBar/pull/31/commits/bd229635af7396f07b110be45d84a18745addab3